spec/dummy/db/migrate/20150713035548_add_function_count_pets.rb

```diff
@@ -0,0 +1,22 @@
+class AddFunctionCountPets < ActiveRecord::Migration
+  def change
+    create_function 'pets_not_empty()', :boolean, <<-FUNCTION, schema: 'public'
+BEGIN
+  IF (SELECT COUNT(*) FROM pets) > 0
+  THEN
+    RETURN true;
+  ELSE
+    RETURN false;
+  END IF;
+END;
```

Comment:
Don't want leading indents when it stores the function in the db. Does Postgres strip out leading whitespace?